### PR TITLE
Web accessibility

### PR DESCRIPTION
**Pull request by marcelosdata**
Using web accessibility easy checks to make create a better experience for all users

### CODE REVIEW

**Comment by marcelosdata** [comment]
# Changes made for this project

## Following the requirements, we check for accessibility issues in the project  📖 

- Page titles: No issues found
- Image text alternatives: Changes on the alternative text on projects 

#

- Text headings: Switch a section Tag to a Div Tag on About-section
- Color contrast: Not changed, following the template but increasing z index on the text of the grid cards and updated bottom borders
- Resize: No issues found
- Interaction: No issues were found
- Forms: Added Labels to Forms elements
- Moving content: No issues found
- Multimedia: No issues found
- The basic structure of the page: No issues were found

File: index.html [hunk 1]
```diff
@@ -112,7 +117,7 @@ <h3>Multi-Post Stories</h3>
             </div>
             <div class="grid-others">
               <div class="projects others mobile-other">
-                <img src="src/Img-Placeholder-proj.svg" alt="#" />
+                <img src="src/Img-Placeholder-proj.svg" alt="Portfolio Image" />
```
Comment (petudeveloper) [inline index.html:120]: To follow every requirement for this project, please [add a link to the corresponding line of code in your pull request changes](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/#:~:text=To%20link%20to%20multiple%20lines,php%23L117%2DL148) for every issue found. This is a good practice that helps communication and that you will surely use in your future projects.

**Review by petudeveloper — CHANGES_REQUESTED** [review]
Hi @marcelosdata @kemar-art, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (marcelosdata) [reply index.html:120]: Hi petudeveloper, Ok I will follow your advice .First I want to ask you if **have you found** any other source on adding a link like this? The instructions on that site are not that intuitive

Comment (petudeveloper) [reply index.html:120]: @marcelosdata I will try to explain myself better, you need to add a link to the corresponding line of code for every correction you made (for every issue). 

e.g. For every **Image text alternatives** (`alt text`) you change, for every **Text headings** correction you made, for every **Form** correction you did, etc. Please add a comment pointing to the correction. 

To do this kindly follow this [link](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/#:~:text=To%20link%20to%20multiple%20lines,php%23L117%2DL148) instructions. For a better explanation look at the images below
1. Go to the `Files changed` tab:
Capture
2. And then click the `+` button to add the comment in the corresponding line of code:
Capture2

File: index.html [hunk 2]
```diff
@@ -130,7 +135,7 @@ <h3>Profesional Art Printing Data</h3>
               </div>
 
               <div class="projects others mobile-other">
-                <img src="src/Img-Placeholder-proj.svg" alt="#" />
+                <img src="src/Img-Placeholder-proj.svg" alt="Portfolio Image" />
                 <div class="info">
                   <h3>Profesional Art Printing Data</h3>
                   <p class="feat-text">
```
Comment (marcelosdata) [inline index.html:141]: Add alternative text to image

File: index.html [hunk 3]
```diff
@@ -147,7 +152,7 @@ <h3>Profesional Art Printing Data</h3>
                 </div>
               </div>
               <div class="projects others mobile-other">
-                <img src="src/Img-Placeholder-proj.svg" alt="#" />
+                <img src="src/Img-Placeholder-proj.svg" alt="Portfolio Image" />
                 <div class="info">
                   <h3>Profesional Art Printing Data</h3>
                   <p class="feat-text">
```
Comment (marcelosdata) [inline index.html:158]: Add alternative text to image

File: index.html [hunk 4]
```diff
@@ -165,7 +170,7 @@ <h3>Profesional Art Printing Data</h3>
               </div>
 
               <div class="projects others mobile-other">
-                <img src="src/Img-Placeholder-proj.svg" alt="#" />
+                <img src="src/Img-Placeholder-proj.svg" alt="Portfolio Image" />
                 <div class="info">
                   <h3>Profesional Art Printing Data</h3>
                   <p class="feat-text">
```
Comment (marcelosdata) [inline index.html:176]: Add alternative text to image

File: index.html [hunk 5]
```diff
@@ -182,7 +187,7 @@ <h3>Profesional Art Printing Data</h3>
                 </div>
               </div>
               <div class="projects others mobile-other">
-                <img src="src/Img-Placeholder-proj.svg" alt="#" />
+                <img src="src/Img-Placeholder-proj.svg" alt="Portfolio Image" />
                 <div class="info">
                   <h3>Profesional Art Printing Data</h3>
                   <p class="feat-text">
```
Comment (marcelosdata) [inline index.html:193]: Add alternative text to image

File: index.html [hunk 6]
```diff
@@ -200,7 +205,7 @@ <h3>Profesional Art Printing Data</h3>
               </div>
 
               <div class="projects others mobile-other">
-                <img src="src/Img-Placeholder-proj.svg" alt="#" />
+                <img src="src/Img-Placeholder-proj.svg" alt="Portfolio Image" />
                 <div class="info">
                   <h3>Profesional Art Printing Data</h3>
                   <p class="feat-text">
```
Comment (marcelosdata) [inline index.html:211]: Add alternative text to image

File: index.html [hunk 7]
```diff
@@ -219,13 +224,16 @@ <h3>Profesional Art Printing Data</h3>
 
               <!-- starts grid desktop version -->
               <div class="projects others desktop-other first-other">
-                <img src="src/Img-Placeholder-clear-v2.svg" alt="#" />
+                <img
+                  src="src/Img-Placeholder-clear-v2.svg"
+                  alt="Portfolio Image"
+                />
                 <div class="info">
```
Comment (marcelosdata) [inline index.html:231]: Add alternative text to image

File: index.html [hunk 8]
```diff
@@ -219,13 +224,16 @@ <h3>Profesional Art Printing Data</h3>
...
+                <img
+                  src="src/Img-Placeholder-clear-v2.svg"
+                  alt="Portfolio Image"
+                />
                 <div class="info">
                   <button class="btn" type="submit">See Project</button>
                 </div>
               </div>
               <div class="projects others desktop-other align-works">
-                <img src="src/works-desktop-2.svg" alt="#" />
```
Comment (marcelosdata) [inline index.html:228]: Add alternative text to image

File: index.html [hunk 9]
```diff
@@ -242,7 +250,7 @@ <h3>Data Dashboard Healthcare</h3>
                 </div>
               </div>
               <div class="projects others desktop-other align-works">
-                <img src="src/works-desktop-3.svg" alt="#" />
```
Comment (marcelosdata) [inline index.html:245]: Add alternative text to image

File: index.html [hunk 10]
```diff
@@ -259,7 +267,7 @@ <h3>Website Portfolio</h3>
                 </div>
               </div>
               <div class="projects others desktop-other align-works">
-                <img src="src/works-desktop-4.svg" alt="#" />
```
Comment (marcelosdata) [inline index.html:262]: Add alternative text to image

File: index.html [hunk 11]
```diff
@@ -276,7 +284,7 @@ <h3>Profesional Art Printing Data More</h3>
                 </div>
               </div>
               <div class="projects others desktop-other align-works">
-                <img src="src/works-desktop-2.svg" alt="#" />
```
Comment (marcelosdata) [inline index.html:279]: Add alternative text to image

File: index.html [hunk 12]
```diff
@@ -293,7 +301,7 @@ <h3>Data Dashboard Healthcare</h3>
                 </div>
               </div>
               <div class="projects others desktop-other align-works">
-                <img src="src/works-desktop-3.svg" alt="#" />
```
Comment (marcelosdata) [inline index.html:296]: Add alternative text to image

File: index.html [hunk 13]
```diff
@@ -317,7 +325,7 @@ <h3>Website Portfolio</h3>
       </section>
 
       <!-- Starts About Section -->
-      <section class="about-container">
+      <div class="about-container">
```
Comment (marcelosdata) [inline index.html:328]: Switch a section Tag to a Div Tag on About-section

File: index.html [hunk 14]
```diff
@@ -394,6 +402,7 @@ <h2 class="form-desc">
             method="POST"
             id="my-form"
           >
+            <label for="name"></label>
             <input
               type="text"
               id="name"
```
Comment (marcelosdata) [inline index.html:408]: Apply Label to the input in form section

File: index.html [hunk 15]
```diff
@@ -402,7 +411,7 @@ <h2 class="form-desc">
               maxlength="30"
               required
             />
-
+            <label for="email"></label>
             <input
               type="email"
               id="email"
```
Comment (marcelosdata) [inline index.html:417]: Apply Label to the input in form section

File: index.html [hunk 16]
```diff
@@ -411,6 +420,7 @@ <h2 class="form-desc">
               required
             />
 
+            <label for="msg"></label>
             <textarea
               name="message"
```
Comment (marcelosdata) [inline index.html:425]: Apply Label to the input in form section

File: stylev2.css [hunk 17]
```diff
@@ -36,11 +36,11 @@ a {
...
-  font-weight: bold;
+  font-weight: 700;
   background-color: var(--primary-color);
   border: solid 1px #fff;
   color: #fff;
-  font-size: 17px;
+  font-size: 2rem;
   line-height: 24px;
   text-align: center;
   border-style: none;
```
Comment (marcelosdata) [inline stylev2.css:46]: Increase font weight and Size

File: stylev2.css [hunk 18]
```diff
@@ -49,11 +49,11 @@ a {
...
 
 .btn:hover {
-  opacity: 0.8;
-  background-color: #fff5e1;
+  opacity: 0.9;
+  background-color: #FFF5E1;
   color: #f55800;
   border: 1px solid #f55800;
-  border-style: none;
+  font-weight: 700;
```
Comment (marcelosdata) [inline stylev2.css]: Diminish Opacity and increase Font Weight

File: stylev2.css [hunk 19]
```diff
@@ -355,7 +355,7 @@ a {
   color: #fff;
   flex: none;
   text-align: start;
-  z-index: 2;
+  z-index: 6;
```
Comment (marcelosdata) [inline stylev2.css:359]: Increase Z Index on element

File: stylev2.css [hunk 20]
```diff
@@ -364,7 +364,7 @@ a {
   font-size: 1.6rem;
   line-height: 2.4rem;
   color: #fff;
-  z-index: 1;
+  z-index: 5;
```
Comment (marcelosdata) [inline stylev2.css:368]: Increase Z index to  better visuals

File: index.html (hunk 1 above)
Comment (marcelosdata) [reply index.html:120]: Awesome, Thanks David

**Review by leolpaz — APPROVED** [review]
Hi @marcelosdata , 

Your project is complete!
Congratulations! 🎉🎉 

To highlight:

- Congratulations on implementing the changes requested by the previous reviewer

Now there is nothing left to do but to merge it! 🏆 





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**